indico/modules/rb/templates/emails/reservations/reminders/finishing_bookings.html

```diff
@@ -29,7 +29,7 @@
 
 
 {% macro _render_dates(reservation) -%}
-    <strong>{{ reservation.start_dt|format_datetime(locale='en_GB') }}</strong> - <strong>{{ reservation.end_dt|format_datetime(locale='en_GB') }}</strong>
+    <strong>{{ reservation.start_dt|format_datetime('EEEE dd/MM/yyyy', locale='en_GB') }}</strong> - <strong>{{ reservation.end_dt|format_datetime('EEEE dd/MM/yyyy', locale='en_GB') }}</strong>
```

Comment:
i'm pretty sure we lose information here since the format you have only includes the data but previously we had the time as well

Reply:
You're right. Also, do you think we should include the full name of the day or just the first three letters (eg. Tue)?